### DISCUSSION TITLE
传奇绳匠法厄同回归

### DISCUSSION THREAD

**Discussion by jikeoff** [share121/inter-knot#2775] [其他]
大家好，我是传奇绳匠法厄同，我的大号因为前段时间躲避治安局的追查，自己注销了，现在风头已经过了，遂决定重出江湖，这个账号日后就作为新的法厄同账号使用，本人因一段时间没接取委托生活比较拮据，现在支持先付款再进行委托，不用担心拿了钱不完成委托，鄙人以法厄同的名号发誓，只要收了钱就一定帮雇主完成委托

**Comment by lixiao189** [comment]
没有 V 我 50 ？

**Comment by I-m-Miss-Lucy-s-dog** [comment]
感覺那個叫妮可的比較有用

**Comment by aizai1173** [comment]
我，Fariy，电费，打钱

**Comment by SnowBall12345678** [comment]
花火大人，你又在恶作剧哦，休息一下好不好！
